test/Spec.hs

```diff
@@ -15,12 +13,17 @@ import IW.Effects.Log (runAppLogIO_)
 import Test.Common (joinSpecs)
 import Test.Core.Issue (issueRoundtripProp)
 import Test.Core.Repo (repoRoundtripProp)
+import Test.Db (dbSpecs)
+import Test.Hspec (Spec, hspec)
+import Test.Hspec.Core.Spec (sequential)
```

Comment:
It was right to put `hspec` modules in the previous section. According to our style guide, imports should be grouped into sections like this:

1. Dependencies from Hackage
2. Dependencies from the project itself

No matter what they share a common prefix. This rule is especially useful when modules share a common name. If I see in imports that `Test.Hspec` is in the first group — I will hoogle it. If I see that `Test.Data` in the last group, I will check local `.cabal` file.

Reply:
Got it!